src/Functions/Theme/theme.js

```diff
@@ -65,6 +71,19 @@ const changeTheme = (document, theme) => {
         changeElementTheme(favorite, "favoriteColor", "color", theme)
         hoverEffect(favorite, themeJSON ? themeJSON[theme].favoriteBackground : defaultThemeJSON[theme].favoriteBackground, themeJSON ? themeJSON[theme].favoriteHoverBackground : defaultThemeJSON[theme].favoriteHoverBackground)
     })
+    document.querySelectorAll(".hover-effect").forEach(obj => {
+        obj.onmouseleave = (e) => {
+            e.target.style.background = getElement("pendriveBackground", theme);
+            e.target.style.borderImage = null;
+        }
+        obj.addEventListener("mousemove", (e) => {
+            const rect = e.target.getBoundingClientRect();
```

Comment:
I think instead of `e.target`, we should just use `obj` instead so that the child element does not affected.

Reply:
Nice idea! LGTM 👍